src/loaders/OMETiffLoader.js

```diff
@@ -43,30 +51,10 @@ export default class OMETiffLoader {
     this.height = this.omexml.SizeY;
     this.tileSize = firstImage.getTileWidth();
     const { SubIFDs } = firstImage.fileDirectory;
-    this.numLevels = this.omexml.getNumberOfImages() || SubIFDs?.length;
+    this.numLevels = SubIFDs?.length || this.omexml.getNumberOfImages();
```

Comment:
This is a small fix needed for images that have multiple Image tags in Bioformats6 since this would have previously displayed the wrong `numLevels` for Bioformats6 pyramids with Image tags.

Reply:
ok